tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Healthchecks.5648.feature

```diff
@@ -5,23 +5,23 @@
 
 @3926
 Scenario: 1. Smtp Server is up
-	Given The Smtp Server is up for ISAPI
-	When the dependency health-check endpoint is hit for ISAPI
+	Given The Smtp Server is up
```

Comment:
Is there a reason why the numbers are different/not matching the 5648

Reply:
I think it is because the scenarios were originally under different story. I'd say to keep them as they are, for documentation sake. What do you think?

Reply:
Hmmm yes, I think you are right actually 😄